src/librustc_trans/debuginfo/mod.rs

```diff
@@ -286,7 +296,7 @@ pub fn create_function_debug_context<'a, 'tcx>(cx: &CrateContext<'a, 'tcx>,
             is_local_to_unit,
             true,
             scope_line as c_uint,
-            FlagPrototyped as c_uint,
+            flags,
```

Comment:
`flags` is initialized to include it:

```
+    let mut flags = FlagPrototyped as c_uint;
```

I'm happy to write this some other way if you think it would be clearer.